Analysis/Stan/mapping_model_generate.stan

```diff
@@ -75,14 +77,17 @@ data {
   int <lower=0, upper=1> censored[K1]; // Whether data is censored in the mapping from observations to location/times
   
   // Mappings
-  int <lower=0, upper=M> map_obs_loctime_obs[K1];    // the observation side of the mapping from observations to location/times
-  int <lower=0, upper=L> map_obs_loctime_loc[K1];    // the location side of the mapping from observations to location/times
-  int <lower=0, upper=L> map_loc_grid_loc[K2];       // the location side of the mapping from locations to gridcells
-  int <lower=0, upper=N> map_loc_grid_grid[K2];      // the gridcell side of the mapping from locations to gridcells
-  int <lower=0, upper=smooth_grid_N> map_smooth_grid[N];    // vector with repeating smooth_grid_N indexes repeating 1:N
+  int <lower=1, upper=M> map_obs_loctime_obs[K1];    // the observation side of the mapping from observations to location/times
```

Comment:
Because Stan objects are indexed starting with 1? For  map_u_loctime_combs_admin_lev, couldn't there be an admin level 0?

Reply:
yes that's it